osu.Game/Overlays/Toolbar/ToolbarClock.cs

```diff
@@ -3,56 +3,84 @@
 
 using osu.Framework.Allocation;
 using osu.Framework.Bindables;
+using osu.Framework.Extensions.Color4Extensions;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
 using osu.Framework.Input.Events;
 using osu.Game.Configuration;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.UserInterface;
 using osuTK;
+using osuTK.Graphics;
 
 namespace osu.Game.Overlays.Toolbar
 {
-    public class ToolbarClock : CompositeDrawable
+    public class ToolbarClock : OsuClickableContainer
     {
         private Bindable<ToolbarClockDisplayMode> clockDisplayMode;
 
+        protected Box HoverBackground;
+        private readonly Box flashBackground;
+        private readonly FillFlowContainer clockContainer;
```

Comment:
Ah right sorry I didn't notice because I copied these over from the standard toolbarbutton, I'll fix this
I'll also be sure to run inspectcode and stuff